CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [3.56.0] - 2024-03-26
 
+### Added
+- `disableStorageApi` config option
+
+### Fixed
+- Subtitle settings not being retained when the UI variant switches
+
```

Comment:
please move to the correct section

Reply:
[Move entry to correct section](https://github.com/bitmovin/bitmovin-player-ui/pull/615/commits/687a242328abdcbb5d8631a0a0a9996481fa1c5e)